src/main/java/frc/robot/subsystems/endEffector/TOFSensor.java

```diff
@@ -0,0 +1,82 @@
+package frc.robot.subsystems.endEffector;
+
+import com.playingwithfusion.TimeOfFlight;
+import com.playingwithfusion.TimeOfFlight.RangingMode;
+
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+public class TOFSensor {
+  public double m_distance_EMA;
+  public double m_dist_SDEV_sq;
+  public String mode = "Short";
```

Comment:
```suggestion
  private String mode = "Short";
```

Probably make this variable private to ensure field integrity

Reply:
Apparently it's not even used in the code.. I'm not sure why it was there. I'll keep it commented out for now.

Reply:
yea let's just leave it there